hieradata_aws/class/mongo.yaml

```diff
@@ -10,3 +10,15 @@ mongodb::server::replicaset_members:
   'mongo-1':
   'mongo-2':
   'mongo-3':
+
+lv:
+  data:
+    pv:
+      - /dev/sdf
```

Comment:
These machines are `m5.large`, which means they use a different device label. Check other hieradata to see what it should be (as I can't remember off the top of my head).

Reply:
https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/device_naming.html and https://aws.amazon.com/amazon-linux-ami/instance-type-matrix/ suggest that M5.large should be HVM EBS backed 64bit, which it suggests for EBS volumes should be /dev/sd[f-p]

Reply:
[Related commit I was talking about](https://github.com/alphagov/govuk-puppet/commit/b1e5add07092de506bee9f0322840fba99acc004)

Reply:
I did see that, I don't know the answer but happy to change it...

Reply:
done